Ginger/GingerCore/Actions/ActLaunchJavaWSApplication.cs

```diff
@@ -305,7 +306,9 @@ public enum ePortConfigType
 
         AutoResetEvent portValueAutoResetEvent;
 
-        public override void Execute()
+        CancellationTokenSource cancellationTokenSource = null; 
+        Task task = null;
```

Comment:
please rename the variables as per naming conventions

Reply:
changes done